### PR TITLE
remove kubevirt related kwok manifests from make cluster-up

### PR DESCRIPTION
**Pull request by alaypatel07**
**What this PR does / why we need it**:

kwok needs kubevirt CRDs to be installed before it can handle KubeVirt related stages. When running make cluster-up kubevirt is still not installed. This is causing bugs in running performance tests. KubeVirt related stages will be created in KubeVirt e2e tests before starting the tests as part of the setup.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
None
```

### CODE REVIEW

**Comment by alaypatel07** [comment]
cc @brianmcarey @xpivarc The setup will be done as part of https://github.com/kubevirt/kubevirt/pull/12117

**Comment by brianmcarey** [comment]
@alaypatel07 what would be the issue with restarting the kwok controller after kubevirt is installed?

**Comment by alaypatel07** [comment]
@brianmcarey restart the kwok controller pod seems like a hack.

Since the e2e test already carries kwok objects like fake nodes https://github.com/kubevirt/kubevirt/pull/12117/files#diff-b1f53ed317ff089a7374611210b0b12f1e5cf165a8cb1748a4cedcd665141805R120, IMO it only makes sense to set up kubevirt related kwok manifests in the e2e tests.

**Comment by xpivarc** [comment]
Would it make sense to deploy only the stages as part of e2e?

File: cluster-up/cluster/kind-1.30/provider.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -e
```
Comment (xpivarc) [inline cluster-up/cluster/kind-1.30/provider.sh:3]: This seems not related to the PR

Comment (alaypatel07) [reply cluster-up/cluster/kind-1.30/provider.sh:3]: good catch, some other local change got mixed into this PR, will remove

**Comment by alaypatel07** [comment]
> Would it make sense to deploy only the stages as part of e2e?

The only other thing being deployed apart from stages is the rbac for kubevirt CRDs. The CRDs are not even registered before rbac is created here since kubevirt is not installed. So I am not sure if it makes sense to just have stages be delayed.

https://github.com/kubevirt/kubevirtci/blob/a087e7e153c809f9c1087cb8d8378b2bc1c6489c/cluster-provision/k8s/1.28/manifests/kwok/kubevirt/rbac/role.yaml

**Comment by xpivarc** [comment]
@alaypatel07 Ok, let's make it work in kubevirt/kubevirt. Please drop the local changes here and we can merge

**Comment by alaypatel07** [comment]
@xpivarc done, PTAL

@Sreeja1725 can you please create the RBAC and kwok stages deleted from here in the e2e test setup https://github.com/kubevirt/kubevirt/pull/12117?

**Review by brianmcarey — COMMENTED** [review]
/approve

**Review by xpivarc — APPROVED** [review]

**Comment by alaypatel07** [comment]
@brianmcarey can you confirm if the kubevirtci image from these changes were pushed in kubevirt repo? I couldnt find any recent bot commits.

cc @Sreeja1725

**Comment by brianmcarey** [comment]
> @brianmcarey can you confirm if the kubevirtci image from these changes were pushed in kubevirt repo? I couldnt find any recent bot commits.
> 
> cc @Sreeja1725

It should be included when this bump (https://github.com/kubevirt/kubevirt/pull/12488) merges
